runtime/battery-station/Cargo.toml

```diff
@@ -173,7 +173,7 @@ runtime-benchmarks = [
     "frame-system-benchmarking",
     "frame-system/runtime-benchmarks",
     "hex-literal",
-    "kusama-runtime?/runtime-benchmarks",
```

Comment:
These seem to be ordered lexicographically, but `"rococo-runtime"` is not sorted in correctly.

Reply:
67906fc019237c8c122e016c8d4f02d0052b4926

Reply:
67906fc019237c8c122e016c8d4f02d0052b4926